amit/index.js

```diff
@@ -0,0 +1,30 @@
+// 1. Array Destructuring
+let fruit = "banana";
+let vegetables = "cucumber";
+let food = ["bread", "cakes", "pizza"];
+let [food1, food2, food3] = food;
+
+console.log(`food1 = ${food1} food2= ${food2} food3= ${food3}`);
+
+// 2 .Object Destructuring
+
+let halloweenCostume = {
+  fran: "witch",
+  alex: "policeman",
+  tommy: "doctor",
+};
+
+let { fran, alex, tommy } = halloweenCostume;
+console.log("costums=", fran, alex, tommy);
+
+// 3  Parameters: Object Destructuring
+let musician = {
+  band: "LRB",
+  vocal: "Baccu",
+  nationality: "Bangladeshi",
+};
+
+let { band, vocal, nationality } = musician;
+
```

Comment:
The last Task ==> Please create a function to display all of the band/musician's data instead of just console.log